VERSION

```diff
@@ -0,0 +1 @@
+0.4.4
```

Comment:
I don't like having yet-another-place with the version number. ``debian/changelog`` should be the only place where such a version number is needed